api/src/controllers/cohesion-center.js

```diff
@@ -232,6 +235,25 @@ router.put("/:id", passport.authenticate("referent", { session: false, failWithE
     await center.save({ fromUser: req.user });
 
     await updateCenterDependencies(center, req.user);
+
+    const IsSchemaDownloadIsTrue = await CohortModel.find({ name: center.cohorts }, "repartitionSchemaDownloadAvailability");
+
+    if (IsSchemaDownloadIsTrue.filter((item) => item.repartitionSchemaDownloadAvailability === true).length) {
```

Comment:
```suggestion
    if (IsSchemaDownloadIsTrue.repartitionSchemaDownloadAvailability === true)) {
```
...

Reply:
ou meme plus simple tu findOne sur la cohort et repartitionSchemaDownloadAvailability : true puis 
```suggestion
    if (IsSchemaDownloadIsTrue) {...}
```